cdap-standalone/pom.xml

```diff
@@ -118,6 +118,16 @@
       <version>${project.version}</version>
     </dependency>
     <dependency>
+      <groupId>co.cask.cdap</groupId>
```

Comment:
Have you try to build SDK and see if these two get included in "lib" directory? Because they shouldn't and I am not sure if the "provided" scope in the "dist" profile is overwrite that or not.